### PR TITLE
Intermediate HTML & CSS Fonts: Remove mention of Google Fonts and Add warning

### PR DESCRIPTION
**Pull request by nikitarevenco**
## Because

Google fonts [violate the GDPR](https://www.cookieyes.com/documentation/features/integrations/google-fonts-and-gdpr/) and as such, should not be used.

## This PR

- Remove mention of google fonts and add warning against using them

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 1]
```diff
@@ -29,7 +29,15 @@ The point of this somewhat ridiculous string of font-families is to try using th
...
-One popular and easy method to get fonts that are *not* installed on a user's computer is to use an online font library like [Google Fonts](https://fonts.google.com/), [Font Library](https://fontlibrary.org/), or the premium, but non-free, [Adobe Fonts](https://fonts.adobe.com/).
+One popular and easy method to get fonts that are *not* installed on a user's computer is to use an online [font library](https://fontlibrary.org/), or the premium, but non-free, [Adobe Fonts](https://fonts.adobe.com/).
+
+### Warning
+
+<div class="lesson-note" markdown="1">
+
+As you look into online font libraries to use you may come across [Google Fonts](https://fonts.google.com/). We strongly advise against using them, as websites that use Google Fonts [violate the GDPR](https://www.cookieyes.com/documentation/features/integrations/google-fonts-and-gdpr/).
+
+</div>
```
[the remark this answers is not in the document]
Comment (nikitarevenco) [reply intermediate_html_css/intermediate_css_concepts/more_text_styles.md:40]: I think its better to leave the article in, it serves as 'proof' to the reader that google fonts violates GDPR. I tried finding an article that doesn't mention how to use google fonts and still stay compliant, but I couldn't - so I've added a change to tell the reader not to follow the instructions on that article as you've mentioned.

File: intermediate_html_css/intermediate_css_concepts/more_text_styles.md [hunk 2]
```diff
@@ -29,7 +29,14 @@ The point of this somewhat ridiculous string of font-families is to try using th
 
 #### Online font libraries
 
-One popular and easy method to get fonts that are *not* installed on a user's computer is to use an online font library like [Google Fonts](https://fonts.google.com/), [Font Library](https://fontlibrary.org/), or the premium, but non-free, [Adobe Fonts](https://fonts.adobe.com/).
+One popular and easy method to get fonts that are *not* installed on a user's computer is to use the online [Font Library](https://fontlibrary.org/), or the premium, but non-free, [Adobe Fonts](https://fonts.adobe.com/).
+
+<div class="lesson-note lesson-note--warning" markdown="1">
+### A warning about Google fonts
+
+As you look into online font libraries to use you may come across [Google Fonts](https://fonts.google.com/). We strongly advise against using them, as websites that use Google Fonts [violate the GDPR](https://www.cookieyes.com/documentation/features/integrations/google-fonts-and-gdpr/). Do not follow the instructions on downloading fonts in that article, as we will be going over it in this lesson.
```
Comment (thatblindgeye) [inline intermediate_html_css/intermediate_css_concepts/more_text_styles.md:37]:
```suggestion
As you look into online font libraries to use you may come across [Google Fonts](https://fonts.google.com/). We strongly advise against using them, as websites that use Google Fonts [violate the GDPR](https://www.cookieyes.com/documentation/features/integrations/google-fonts-and-gdpr/). Do not follow the instructions in that article on how to make Google fonts compliant, as we will be going over downloading fonts in this lesson.
```

**Review by thatblindgeye — APPROVED** [review]
Looks good! Had a nit below, more personal preference than anything, but let me know what you think makes the most sense to you and I'd be good either way to merge this in.

**Comment by nikitarevenco** [comment]
> Looks good! Had a nit below, more personal preference than anything, but let me know what you think makes the most sense to you and I'd be good either way to merge this in.

Great, I think your suggestion improves the clarity & wording. I added it.

**Pull request merged**